src/models/piranha_arguments.rs

```diff
@@ -11,19 +11,19 @@ Copyright (c) 2022 Uber Technologies, Inc.
  limitations under the License.
 */
 
-use crate::utilities::read_toml;
-
 use super::{
   default_configs::{
     default_cleanup_comments, default_cleanup_comments_buffer,
     default_delete_consecutive_new_lines, default_delete_file_if_empty, default_dry_run,
-    default_global_tag_prefix, default_input_substitutions, default_language,
-    default_name_of_piranha_argument_toml, default_number_of_ancestors_in_parent_scope,
-    default_path_to_codebase, default_path_to_configurations, default_path_to_output_summaries,
-    default_piranha_language, default_substitutions,
+    default_global_tag_prefix, default_name_of_piranha_argument_toml,
+    default_number_of_ancestors_in_parent_scope, default_path_to_codebase,
+    default_path_to_configurations, default_path_to_output_summaries, default_piranha_language,
+    default_substitutions, GO, JAVA, KOTLIN, PYTHON, SWIFT, TSX, TYPESCRIPT,
   },
   language::PiranhaLanguage,
 };
+use crate::utilities::{parse_key_val, read_toml};
+use clap::builder::TypedValueParser as _;
```

Comment:
Removed. It was part of the tutorial, I copy pasted (didnt update) [Ref](https://doc.rust-lang.org/reference/items/use-declarations.html#:~:text=Underscore%20Imports,may%20conflict%20with%20another%20symbol). They do this so that when folks copy paste into their snippets in their own code base they dont get namespace conflicts

Reply:
Ok, do we have any sort of check about unused imports / import order and formatting in Rust? I see a lot of bikeshedding on these import blocks (a fix for a different PR at any rate, just think aloud here).

Reply:
Actually, rust clippy takes care of unused imports. It is not aggressive about ordering the imports , but it has many settings which can fix the things you suggest. I will look into it (on some slow afternoon :P )